docs/docs/working-with-video.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Working With Video*
```

Comment:
I don't think the asterisk should be included here — it should only be in the sidebar YML to trigger the "stub" styling. (I think.)

Reply:
This is correct! Titles stay clean, the yaml gets updated with an asterisk!